pkg/autoscaler/k8sclient/k8sclient_test.go

```diff
@@ -76,6 +76,76 @@ func TestGetScaleTarget(t *testing.T) {
 	}
 }
 
+func TestGetScaleTarget(t *testing.T) {
```

Comment:
nit: `TestGetScaleTargets` so it is obvious this covers the multi-target scenario.